bundles/org.openhab.binding.networkupstools/README.md

```diff
@@ -10,18 +10,16 @@ This binding lets you integrate NUT servers with openHAB.
 ## Supported Things
 
 The binding can connect to multiple NUT instances.
-The thing is a `ups` thing.
-The thing supports a number of nut features out-of-the-box.
-And supports the option to configure additional channels to get other NUT variables.
-The thing queries the NUT server for the status of the UPS approximate every 3 seconds.
-And updates the status when a change happens.
-When the UPS status changes it updates all other linked channels.
-Besides the UPS status change update the channels are updated at the user configured refresh time.
-
-Some nut variables are static in nature and are not suited for a channel.
-Some of these could be updated, like of firmware version.
-Therefore the properties are updated with a 1 hour frequency.
-The following nut variables are are read and added to the thing as properties:
+The thing is an `ups` thing.
+The thing supports a number of NUT features out-of-the-box and supports the option to configure additional channels to get other NUT variables.
+The thing queries the NUT server for the status of the UPS approximate every 3 seconds and updates the status when a change happens.
+When a change of the UPS status happens it will query the NUT server to update all linked channels.
+Outside the status change updates, all linked channels are updated at the user configured refresh time.
+
+Some NUT variables are static in nature and are not suited for a channel.
+Some of these could change, like of firmware version.
+Therefore these properties are updated with a 1 hour frequency.
+The following NUT variables are are read and added to the thing as properties:
```

Comment:
"The following NUT variables are are read ..."

"are" occurs twice. I missed that on my initial review.